### DISCUSSION TITLE
Why table doesn't have super params?

### DISCUSSION THREAD

**Discussion by yahu1031** [serverpod/serverpod#1371] [Q&A] [answered]
<img width="1352" alt="Screenshot 2023-09-16 at 12 34 56 AM" src="https://github.com/serverpod/serverpod/assets/35523357/109e3aaf-5a28-469f-b1e8-808b1a954fc3">
Getting this issue. Why? @Isakdl any idea?

**Comment by Isakdl** [comment]
Hmm are you using the cli from the main branch but serverpod dependency from pub.dev?

**Reply by yahu1031** [reply, marked as answer]
Nope, got it a glitch by ide that points to pub caches somehow.
